tests/test_flair_configs.py

```diff
@@ -0,0 +1,150 @@
+from dataclasses import fields
+from tempfile import TemporaryDirectory
+from typing import Any, Dict, Tuple
+
+import datasets
+import flair
+import pytest
+import torch
+from flair.data import Corpus
+from numpy import typing as nptyping
+
+from embeddings.config.flair_config import (
+    FlairSequenceLabelingAdvancedConfig,
+    FlairSequenceLabelingBasicConfig,
+    FlairSequenceLabelingConfig,
+)
+from embeddings.data.data_loader import HuggingFaceDataLoader
+from embeddings.data.dataset import Dataset
+from embeddings.pipeline.evaluation_pipeline import (
+    FlairSequenceLabelingEvaluationPipeline,
+    ModelEvaluationPipeline,
+)
+from embeddings.pipeline.preprocessing_pipeline import PreprocessingPipeline
+from embeddings.transformation.flair_transformation.column_corpus_transformation import (
+    ColumnCorpusTransformation,
+)
+from embeddings.transformation.flair_transformation.downsample_corpus_transformation import (
+    DownsampleFlairCorpusTransformation,
+)
+from embeddings.transformation.flair_transformation.split_sample_corpus_transformation import (
+    SampleSplitsFlairCorpusTransformation,
+)
+from embeddings.utils.flair_corpus_persister import FlairConllPersister
+
+
+@pytest.fixture(scope="module")
+def result_path() -> "TemporaryDirectory[str]":
+    return TemporaryDirectory()
```

Comment:
```suggestion
@pytest.fixture(scope="module")
def result_path(tmpdir_factory: TempdirFactory) -> Path:
    path = tmpdir_factory.mktemp(__name__)
    return Path(path)
```

Reply:
I get FileNotFoundError with this solution

Reply:
ok this is now fixed